firstvoices/backend/management/commands/copy_site.py

```diff
@@ -175,9 +176,9 @@ def copy_audio_and_speakers_and_return_map(source_site, target_site, logger):
             audio_map[source_audio_id] = target_audio_id
 
             audio.id = target_audio_id
-            # To circumvent checks added to media models to prevent modification of original file
+            # To circumvent certain conditionals that were originally added to prevent modification of the original file
```

Comment:
Minor: isn't this actually here to make sure it doesn't get confused when checking whether we need to generate thumbnails?

Reply:
yep, my bad. Confused this with the file model check. Will update it